src/components/ConversationSettings/DangerZone.vue

```diff
@@ -14,9 +14,26 @@
 				<p class="danger-zone__hint">
 					{{ t('spreed', 'Once a conversation is left, to rejoin a closed conversation, an invite is needed. An open conversation can be rejoined at any time.') }}
 				</p>
-				<NcButton type="warning" @click="leaveConversation">
+				<NcButton type="warning" @click="toggleShowLeaveConversationDialog">
 					{{ t('spreed', 'Leave conversation') }}
 				</NcButton>
+				<NcDialog class="danger-zone__dialog"
+					:open.sync="isLeaveConversationDialogOpen"
+					:name="t('spreed','Leave conversation')"
+					:message="leaveConversationDialogMessage"
+					container=".danger-zone">
+					<template #actions>
+						<NcButton v-if="supportsArchive && !conversation.isArchived" type="secondary" @click="toggleArchiveConversation">
+							{{ t('spreed', 'Archive instead') }}
```

Comment:
Explain what it changes?

Reply:
Copy-paste from archive toggle should be fine here as well

Reply:
Nah, too much text. Placing 'Archive' option in Danger zone should be enough to explain for both
![image](https://github.com/user-attachments/assets/7e5905be-566a-4920-995f-d5e12cf26628)